specification/columns/servicesubcategory.md

```diff
@@ -0,0 +1,123 @@
+| Business Applications     | Other (Business Applications)     | Business Applications services that do not fall into one of the defined subcategories.                               |
+| Compute                   | Containers                        | Management and orchestration of containerized compute platforms.                                                     |
+| Compute                   | End User Computing                | Virtualized desktop infrastructure and device / endpoint management.                                                 |
+| Compute                   | Quantum                           | Resources and simulators that leverage the principles of quantum mechanics.                                          |
+| Compute                   | Serverless Compute                | Enablement of compute capabilities without provisioning or managing servers.                                         |
+| Compute                   | Virtual Machines                  | Computing environments created by abstracting resources from physical hardware.                                      |
+| Compute                   | Other (Compute)                   | Compute services that do not fall into one of the defined subcategories.                                             |
+| Databases                 | Cache Databases                   | Low-latency and high-throughput access to frequently accessed data.                                                  |
+| Databases                 | Data Warehouses                   | Big data storage and querying capabilities.                                                                          |
+| Databases                 | Ledger Databases                  | Immutable and transparent databases to record tamper-proof and cryptographically secure transactions.                |
+| Databases                 | NoSQL Databases                   | Unstructured or semi-structured data storage and querying capabilities.                                              |
+| Databases                 | Relational Databases              | Structured data storage and querying capabilities.                                                                   |
+| Databases                 | Time Series Databases             | Time-stamped data storage and querying capabilities.                                                                 |
+| Databases                 | Other (Databases)                 | Database services that do not fall into one of the defined subcategories.                                            |
+| Developer Tools           | Developer Platforms               | Integrated environments that facilitate multiple development needs.                                                  |
+| Developer Tools           | Deployment                        | Tools and services that support deploying code for software and systems.                                             |
```

Comment:
Want to avoid acronyms if possible.  Don't love `Integration and Deployment` because it starts with "Integration", which is a category name.  How do you feel about `Continuous Integration and Deployment`?
```suggestion
| Developer Tools           | Continuous Integration and Deployment                        | CI/CD tools and services that support building and deploying code for software and systems.           |
```

Reply:
While I prefer the simplicity of "Deployment", I'm fine with this. 

Is there anything that falls into the "Deployment" bucket that is not explicitly CI/CD? (Side note: Copilot prefers "Continuous Integration and Deployment" as a more holistic name.)

Reply:
Cool!  Committing and resolving this one.